spec/models/user_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  describe "Associations" do
+    it { should have_many(:posts) }
+  end
+
+  it "can not create account without email" do
+    user = build(:user, email: nil)
+    user.valid?
+    expect(user.errors[:email]).to include("can't be blank")
```

Comment:
This is great! 

As you can imagine, you will end up writing tests just like this all the time. Since you have shoulda-matchers, save yourself some of that time and see if you can find a way to test presence validations using one of the helpers that it provides.

Reply:
See the example in [this section of the docs](https://github.com/thoughtbot/shoulda-matchers#usage) for how shoulda-matchers can make this a little easier.